frontend/packages/shared/src/navigation/PackagesRouter/PackagesRouter.test.ts

```diff
@@ -0,0 +1,89 @@
+import { PackagesRouter } from './PackagesRouter';
+
+const mockOrg: string = 'org';
+const mockApp: string = 'app';
+
+describe('PackagesRouter', () => {
+  describe('constructor', () => {
+    it('should initialize app and org properties', () => {
+      const packagesRouter = new PackagesRouter({ org: mockOrg, app: mockApp });
+
+      expect(packagesRouter['app']).toEqual(mockApp);
+      expect(packagesRouter['org']).toEqual(mockOrg);
+    });
+
+    it('should default to empty strings if app and org are not provided', () => {
+      const routerWithoutParams = new PackagesRouter({});
+      expect(routerWithoutParams['app']).toEqual('');
+      expect(routerWithoutParams['org']).toEqual('');
+    });
+  });
+
+  describe('navigateToPackage', () => {
+    it('should navigate to the correct URL without subUrl', () => {
```

Comment:
Should we change the description to the following:

```suggestion
    it('should navigate to the "editor/overview" page when the location parameter is set to "editorOverview."', () => {
```